src/MOI/MOI_wrapper.jl

```diff
@@ -1484,11 +1484,18 @@ function MOI.delete(
     return
 end
 
+function _fix_variable_value(model, info, value)
+    Lib.XPRSchgbounds(model.inner, Cint(1), Ref(Cint(info.column-1)), Ref(UInt8('B')), Ref(value))
```

Comment:
```suggestion
    Lib.XPRSchgbounds(model.inner, Cint(1), Ref(Cint(info.column-1)), Ref(Cchar('B')), Ref(value))
```

Reply:
This change causes an error: `unsafe_convert(::Type{Ptr{UInt8}}, ::Base.RefValue{Int8})`

Reply:
Urgh. Okay. The C documentation says it should be a `Cchar`: https://www.fico.com/fico-xpress-optimization/docs/latest/solver/optimizer/HTML/XPRSchgbounds.html. I don't know how the `lib.jl` was generated. Perhaps the header file is actually a UInt8 instead of Int8.